lib/backup/azremote/azblob.go

```diff
@@ -434,3 +454,15 @@ func cleanDirectory(dir string) string {
 
 	return dir
 }
+
+// maybeSetMetadata sets metadata for the blob if metadata is not empty.
```

Comment:
why this func has `maybe` prefix? It seems like `maybe`-iness is already expected due to func signature as it could return an error